app/components/map-toolbar.js

```diff
@@ -43,6 +43,14 @@ export default Component.extend({
     return choroplethConfigs.find(d => d.id === mode).legendTitle;
   }),
 
+  insignificantLegendLabel: computed('choroplethMode', function() {
+    const { choroplethMode: mode } = this.getProperties('choroplethMode');
```

Comment:
You can just do `const mode = this.get('choroplethMode');` ?

Reply:
fixed